app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -271,6 +271,9 @@ interface ProductInterface @typeResolver(class: "Magento\\CatalogGraphQl\\Model\
     websites: [Website] @doc(description: "An array of websites in which the product is available") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Websites")
     product_links: [ProductLinksInterface] @doc(description: "An array of ProductLinks objects") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\ProductLinks")
     media_gallery_entries: [MediaGalleryEntry] @doc(description: "An array of MediaGalleryEntry objects") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\MediaGalleryEntries")
+    related_products: RelatedProduct @doc(description: "RelatedProduct") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Related")
```

Comment:
Looks like resolver should be `Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Related\\RelatedProducts`
Why do we need one more layer?
The same for another types of related products